app/lib/search/search_service.dart

```diff
@@ -244,10 +244,13 @@ class ServiceSearchQuery {
     return map;
   }
 
+  late final effectiveOrder = parsedQuery.order ?? order;
```

Comment:
Can this have some documentation? what is `order` and `effectiveOrder`, do we ever need both?

Reply:
We need it until the URL-parameter order is removed from the supported query.